js/app.js

```diff
@@ -0,0 +1,79 @@
+// Section 1
+
+// 1. Concatenation is combining strings. Interpolation is using a variable to represent a value inside a string.
+	// concatenation: console.log("This " + "is " + "a " + "new " + "string.");
+	// interpolation: 
+		// let word = "new";
+		// console.log("This " + "is " + "a " + word + " " + "string.");
+
+// 2. D.R.Y. stands for Don't Repat Yourself. Repeated code is inefficient, so we use loops to avoid repetitive code. 
+
+
+
+// Section 2
+
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+// console.log(a <= b);
+// console.log(c >= d);
+// console.log('Name' === 'Name');
+// console.log(a + b === c);
+// console.log(a * a === d);
+// console.log(e === 'Kevin');
+// console.log(48 == '48');
+
+
+
+// Section 3
+
+// Infinite loop 1: This is not an infinite loop because it doesn't specify what has to be true. 
+
+// Infinite loop II: This is an infinite loop because you can't change the value of a constant, so runProgram = false will give an error and the loop will repeat infinitely because runProgram = true. 
```

Comment:
nice job catching that it should throw an error.  the error actually stops the execution.  so additionally, conceptually it is not an infinite loop, even though that's not the reason it doesn't keep repeating.